webview-ui/src/components/mcp/configuration/tabs/installed/server-row/ServerRow.tsx

```diff
@@ -1,214 +1,22 @@
+import { McpServer } from "../../../../../../../../src/shared/mcp"
```

Comment:
If we were to add path alias this could like something like 
```suggestion
import { McpServer } from "@shared/mcp"
```

Reply:
@saito-sv Yeah will be adding those